### PR TITLE
Component reviews

### LINKED ISSUE #13: Components/review card

### LINKED ISSUE #14: Components/review stars

### PR DESCRIPTION
**Pull request by oskarprzybylski23**
## Description
Added following components:

- review starts (not used but kept for the future in case needed)
- TestimonyCard
- Carousel (from Shadcn)
- CarouselTestimony - utilising Shadcn carousel to create carousel with testimony cards

Moreover new library ```embla-carousel-autoplay``` has been installed
I've also added the new carousel to ``Home`` and style a bit to organise layout.
Changed export name of quote carousel to ``CarouselQuote`` to avoid confusion and conflict with other carousel types

Carousel data is taken from ``testimonies.ts`` file.

image

## Related Issue
closes #13 
closes #14 
## How Can This Be Tested?

``npm install`` > ``npm run dev`` > go to home and see

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings, including `` npm run build ``
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)

### CODE REVIEW

**Review by halimahexe — CHANGES_REQUESTED** [review]
This looks great but I think the display font is a bit hard to read. Could we switch so the sans-serif font is used for the content of the testimonials and the serif font for the name?

Also, the size of these cards gets massive at smaller screen sizes. See my note for where this might be changed.

Good job!

File: src/components/CarouselTestimonial/CarouselTestimonial.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				className='w-full max-w-7xl'
+			>
+				<CarouselPrevious />
+				<CarouselContent className='-ml-2 md:-ml-20'>
+					{testimonials.map((testimonial, index) => (
+						<CarouselItem
+							key={index}
+							className='pl-2 md:pl-20  md:basis-1/2 lg:basis-1/3'
+						>
+							<TestimonialCard
```
[the remark this answers is not in the document]
Comment (oskarprzybylski23) [reply src/components/CarouselTestimonial/CarouselTestimonial.tsx:51]: Yes, I've added some more responsivity for smaller screens:

image

[the remark this answers is not in the document]
Comment (oskarprzybylski23) [reply src/components/CarouselTestimonial/CarouselTestimonial.tsx:51]: Changed fonts too. Lmk if that's better. Thanks

**Review by halimahexe — APPROVED** [review]
LGTM!

**Pull request merged**